src/cli.ts

```diff
@@ -14,6 +14,11 @@ const cli = meow(`
 			Options
 				-c --context  Your AsyncAPI specification
 				-w --watch Enable watchMode (not implemented yet)
+		context
+			current                show the current set context
+			list     			   list all the contexts
+			remove  			   remove the current set context 
```

Comment:
how about making it optional that you can also remove specific context by providing its name?